client/boot/index.js

```diff
@@ -369,6 +369,10 @@ function reduxStoreReady( reduxStore ) {
 		require( 'lib/rubberband-scroll-disable' )( document.body );
 	}
 
+	if ( config.isEnabled( 'dev/test-helper' ) && document.querySelector( '.environment.is-tests ul.active-tests' ) ) {
+		require( 'lib/abtest/test-helper' )( document.querySelector( '.environment.is-tests ul.active-tests' ) );
```

Comment:
It feels a bit weird to have half of the badge backed there, and this section as a react component. Did you have trouble getting all the needed information from the jade index.js controller?